Feature/invoice forms cleanup

Srinivas26k opened the pull request:
solved the issue #7 
 - Integrateg Daisy UI components for a clean, professional design.
 - Used KokonutUI for accessible, ready-to-use elements.
 - Refine typography, spacing, and color palettes for better readability.

<img width="1920" height="965" alt="image" src="https://github.com/user-attachments/assets/f84ef614-bb17-44aa-83fe-760b2dbaa8fd" />

Srinivas26k committed:
Add new favicon images for the application

- Introduced a 16x16 pixel favicon image to enhance branding.
- Added a 32x32 pixel favicon image for improved visibility on high-resolution displays.
- Changed the File structre according to the industry standard and agile method for future enhancements.

Srinivas26k committed:
feat: Add client and invoice forms with item management

- Added ClientInfoFormNew component for client information input.
- Introduced InvoiceSummaryNew component to display invoice summary.
- Created ItemModal component for adding and editing line items.
- Implemented LineItemsTableNew component to manage line items.
- Developed InvoicePreviewPanel component for invoice preview.
- Updated design.html for improved layout and styling.
- Integrated daisyUI for enhanced UI components.
- Updated Tailwind CSS configuration for daisyUI support.
- Modified postcss.config.mjs for plugin configuration.
- Updated package.json to include daisyUI and updated Tailwind CSS versions.

Srinivas26k committed:
Refactor: Remove deprecated components and consolidate invoice forms

- Deleted ClientInfoFormNew.tsx and InvoiceSummaryNew.tsx as they were backups.
- Removed old LineItemsTable.tsx and corrupted InvoiceSummary.tsx, replacing them with updated versions.
- Cleaned up tailwind.config.ts as it was a duplicate.

Vaibhav-Samdani commented:
1. Use billForge logo given in the public folder.
2. Is it responsive for all devices?

Srinivas26k committed:
feat: Make entire project responsive for all device sizes

- Replace material icons logo with actual logo image from public folder
- Add responsive header with sticky positioning and adaptive button sizes
- Implement mobile-first responsive layout for main page
- Create mobile card view for line items table on small screens
- Add responsive grid layouts for business and client info forms
- Update typography and spacing for different screen sizes
- Enhance button responsiveness with size variants
- Optimize table cell padding and font sizes for mobile
- Improve touch targets and spacing for mobile usability

Srinivas26k commented:
> 1. Use billForge logo given in the public folder.
> 2. Is it responsive for all devices?

* I’ve used the BillForge logo from the `/public` folder as requested.
* Regarding responsiveness — that wasn’t part of my assigned scope. 
My original task was:
  1. Integrate DaisyUI components for a clean, professional design.
  2. Use KokonutUI for accessible, ready-to-use elements.
  3. Refine typography, spacing, and color palettes for better readability.

That said, I’ve gone ahead and fixed the responsiveness as well.
Since the PR now goes beyond the original requirements, I’d like to request upgrading this to a **Level-2** tag.

*Attaching a screenshot for reference:*
<img width="1920" height="967" alt="image" src="https://github.com/user-attachments/assets/230aef0f-4525-44b5-bbc7-238e8183f976" />

Vaibhav-Samdani commented:
do you change the pdf template ?

Srinivas26k commented:
I want to this close this PR and then work on the templates feature, as its another issue completely.

Vaibhav-Samdani commented:
input forms are also responsive?

Srinivas26k commented:
> input forms are also responsive?

Yes , AS the Issue I raised Ingerating the Components from ShadCn and DaisyUI I integrated with responsiveness also.
<img width="1920" height="936" alt="image" src="https://github.com/user-attachments/assets/adcd9b01-c39b-4df0-9533-efa98c3b0299" />
<img width="1920" height="936" alt="image" src="https://github.com/user-attachments/assets/a07607d3-45f5-4dfb-bddd-7e2a0782b81d" />
<img width="1920" height="936" alt="image" src="https://github.com/user-attachments/assets/ab19f7f0-35fa-4fa9-a4c5-37273a2235f4" />

----
Once you merge I will create a new pull request for adding invoices templates.(Which makes me update my local branch with your latest branch, so that no issues happen while merging with other PR)

Vaibhav-Samdani commented:
for each new feature you should use a new branch of the main branch.

Vaibhav-Samdani commented:
and please revert the last invoice template.

Vaibhav-Samdani commented:
check #20
